packages/shared-state-bat_links_info/files/usr/bin/shared-state-publish_bat_links_info

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/lua
+
+--! LibreMesh
+--! Copyright (C) 2023  Javier Jorge <jjorge@inti.gob.ar>
+--! Copyright (C) 2023  Asociación Civil Altermundi <info@altermundi.net>
+--!
+--! This program is free software: you can redistribute it and/or modify
+--! it under the terms of the GNU Affero General Public License as
+--! published by the Free Software Foundation, either version 3 of the
+--! License, or (at your option) any later version.
+--!
+--! This program is distributed in the hope that it will be useful,
+--! but WITHOUT ANY WARRANTY; without even the implied warranty of
+--! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--! GNU Affero General Public License for more details.
+--!
+--! You should have received a copy of the GNU Affero General Public License
+--! along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+local JSON = require("luci.jsonc")
+local utils = require('lime.utils')
+local network = require ("lime.network")
+
+local hostname = utils.hostname()
+
+function get_bat_links_info()
+	local batneights_obj={}
+	local batlinks_obj={}
+	local batlinks = utils.unsafe_shell("batctl oj")
+	batlinks_obj = JSON.parse(batlinks)
+
+
+	local batneights = utils.unsafe_shell("batctl nj")
+	batneights = string.gsub(batneights,"neigh_address","dst_mac")
+	batlinks = string.gsub(batlinks,"orig_address","src_mac")
+	batneights = string.gsub(batneights,"hard_ifname","iface")
+	batneights_obj = JSON.parse(batneights)
+
+	for key,neight_value in pairs (batneights_obj) do
+		local macparts = network.get_mac(neight_value.iface)
+		local src_macaddr = table.concat(macparts,":")
+		neight_value.hard_ifindex=nil
+		neight_value.src_mac=src_macaddr
+		for key,link_value in pairs (batlinks_obj) do
+			if link_value.hard_ifname == neight_value.iface and 
+			link_value.neigh_address== link_value.orig_address and  
+			link_value.neigh_address== neight_value.dst_mac then
+				neight_value.tq = link_value.tq
```

Comment:
Let's add a comment explaining that `tq` stands for Batman Transmit Quality

Reply:
sure !